.buildkite/pipeline_pr.py

```diff
@@ -25,7 +25,7 @@ def get_changed_files(branch):
 args = COMMON_PARSER.parse_args()
 
 step_style = {
-    "command": "./tools/devtool -y test -- ../tests/integration_tests/style/",
```

Comment:
Can we make it so that this is the default? i.e. If we want to run this tests, we would have to explicitly say `-m optional`. To see how to do that, look at `conftest.py::pytest_collection_modify_items`.

Maybe `benchmarks` is a more descriptive marker name, or, why not just re-using the `nonci` marker?

Reply:
I was thinking that the marker should be general enough to allow for any kind of "optional" test we might want to add in the future (maybe some short running perf tests that we just use to collect numbers, or some of the other tests that currently rely on us adjusting numbers). That's why I didnt go with "benchmarks". As for nonci, since the optional pipeline runs all tests that have the marker, reusing it would cause it to also run all nightly perf tests :(